crates/biome_cli/src/commands/clean.rs

```diff
@@ -0,0 +1,10 @@
+pub use crate::commands::daemon::biome_log_dir;
```

Comment:
Do we need this `pub`?
```suggestion
pub use crate::commands::daemon::biome_log_dir;
```